File: workflows/examples/workflow_types_json.rb
require "net/http"
require "uri"
require "openssl"

uri = URI.parse("https://test2-lairdassessors.swiftcase.co.uk/api/v2/YOURAPIKEY/workflows.json")
http = Net::HTTP.new(uri.host, uri.port)
http.use_ssl = true

request = Net::HTTP::Get.new(uri.request_uri)
request["User-Agent"] = "Laird Assessors API Example"
request["Connection"] = "close"
request["Accept-Encoding"] = "gzip, deflate"
request["Accept"] = "*/*"
response = http.request(request)

puts "Status code: "+response.code
puts "Response body: "+response.body

File: workflows/examples/workflow_types_json.php
<?php
$YOURAPIKEY = "";
$req = curl_init("https://test2-lairdassessors.swiftcase.co.uk:443/api/v2/".$YOURAPIKEY."/workflows.json");
curl_setopt($req, CURLOPT_RETURNTRANSFER, true);
curl_setopt($req, CURLOPT_HTTPHEADER, array("User-Agent: Laird Assessors API Example","Connection: close","Accept-Encoding: gzip, deflate","Accept: */*"));
$result = curl_exec($req);

echo "Status code: ".curl_getinfo($req, CURLINFO_HTTP_CODE)."\n";
echo "Response body: ".$result."\n";

curl_close($req);

?>
